spec/controllers/posts_controller_spec.rb

```diff
@@ -2502,16 +2502,16 @@ def restore(reply)
       expect(flash[:success]).to eq("Post deleted.")
     end
 
-    it "deletes PostAuthors" do
+    it "deletes Post::Authors" do
```

Comment:
Not sure if this was intended or overeager find/replace but is fine either way, just stood out to me.

Reply:
Both, find/replace found it but I did intentionally decide that if the message is specifying the name of the class then I would specify the _correct_ name of the class at least.